NEWS.md

```diff
@@ -1,3 +1,7 @@
+# nflplotR (development version)
+
+* nflplotR will internally cache images used in all geoms and elements. The cache behavior can be controlled by setting `options("nflplotR.cache")` to one of `"memory"`, `"filesystem"`, or `"off"`. It is possible to clear the cache with the new function `.clear_cache()`. This functionality added the dependencies cachem, memoise and rappdirs.
```

Comment:
(which were already depended on via nflreadr so no real new dependencies!)

Reply:
That's why I actually did it, because it doesn't make a difference. But I have to import these package explicitely now which is the reason why I consider it a new dependency.